.circleci/verbatim-sources/header-section.yml

```diff
@@ -161,6 +167,86 @@ install_doc_push_script: &install_doc_push_script
       echo "Skipping push due to dry_run"
     fi
 
+    popd
+
+    # ======================== Building PyTorch C++ API Docs ========================
```

Comment:
I'm going to assume that you pulled all of the following from the chronos job?

Reply:
Yes I did, and I also verified that it works in CircleCI.